test/unit/browser/invalidation_mixin_test.dart

```diff
@@ -36,9 +36,9 @@ void main() {
         expect(thing.invalid, isTrue);
 
         // ignore: unawaited_futures
-        onValidation.then(expectAsync((ValidationStatus status) {
+        onValidation.then(expectAsync1((ValidationStatus status) {
           expect(status, equals(ValidationStatus.cancelled));
-        }, count: 1));
```

Comment:
Ha. @todbachman-wf suggested in a previous PR that I remove redundant counts. I obliged.